### DISCUSSION TITLE
NDVI and drought - MODIS data for Necedah, Wisconsin Location

### DISCUSSION THREAD

**Discussion by DharaniSureshBabu** [cu-esiil-edu/2024-data-short-course#70] [Q&A]
@eculler @nquarder 
Hi Elsa and Nate, 
Could you please help me to download NDVI and drought MODIS data for this location, 44.216612, -90.076390 located at Necedah, Wisconsin? This will be very helpful for my research. Thank you for your time and support!

**Comment by nquarder** [comment]
Okay @DharaniSureshBabu here is some code you can use to bring in Juneau County, WI:

#Open Wisconsin County Data
wi_counties_url = 'https://www2.census.gov/geo/tiger/TIGER2018/COUSUB/tl_2018_55_cousub.zip'
wi_counties_gdf = gpd.read_file(wi_counties_url)
wi_counties_gdf

#Select out Juneau County
juneau = wi_counties_gdf[wi_counties_gdf["NAME"] == "Juneau"]
juneau

**Reply by DharaniSureshBabu** [reply]
Thanks much, Nate! @nquarder
